### ISSUE TITLE
Show English dataset description at reconciliation page 

### ISSUE THREAD

**Issue by rschalkrce** [netwerk-digitaal-erfgoed/network-of-terms-demo#835]
Currently the dataset overview at the [reconciliation page](https://termennetwerk.netwerkdigitaalerfgoed.nl/reconciliation) shows Dutch descriptions only. This is probably because there are no English descriptions in the [dataset catalogue](https://github.com/netwerk-digitaal-erfgoed/network-of-terms/tree/master/packages/network-of-terms-catalog/catalog/datasets) in the network-of-terms repo. 

Can we show English dataset descriptions at the reconciliation page, possibly by we adding English labels to the dataset catalogue?

**Comment by ddeboer** [comment]
Yeah, no problem. Just to be sure: is there actually user demand for this? The NoT is geared towards Dutch heritage data and most users are Dutch.
